sql/sys_vars.cc

```diff
@@ -741,6 +741,29 @@ static Sys_var_charptr_fscs Sys_character_sets_dir(
        READ_ONLY GLOBAL_VAR(charsets_dir), CMD_LINE(REQUIRED_ARG),
        DEFAULT(0));
 
+static bool check_engine_supports_temporary(sys_var *self, THD *thd, set_var *var)
+{
+  if (!var->value || var->value->is_null())
+    return false;
+  String str, *res;
```

Comment:
This is just a style nitpick. Let's move both line 748 and line 750 at the beginning of the function. After Line 745.

Reply:
Thanks for the review @cvicentiu and for debugging sessions resulting in better quality and learning the new things.